common/tools/dev-tool/src/commands/run/extract-api.ts

```diff
@@ -190,7 +176,7 @@ export default leafCommand(commandInfo, async () => {
 
       const updatedConfigObject: IConfigFile = {
         ...extractorConfigObject,
-        dtsRollup: newDtsRollupOptions,
+        dtsRollup: { enabled: false }, // disable dtsRollup for ESM/tshy packages
```

Comment:
I wonder if we should also / instead update the api extractor json to disable dtsRollup and then leave this unchanged (i.e. not set it to false)? That might be clearer than setting the value here (although it'll require one NO_CI commit)

Reply:
Yes, that should be cleaner! We probably still need to check the existence of dtsRollupOptions here to prevent adding a new one again.

Reply:
or just don't pass "dtsRollup" to api-extractor at all.

Reply:
Since we still need some changes here.  How about we merge this and later followed by either 
- a codemod in migration and a NO_CI commit to remove dts rollup options from api-extractor.json for migrated packages

or

- at the end of migration apply a one-time NO_CI commit to remove the dts rollup options.

Reply:
Sure! This isn't blocking so however you think makes the most sense 👍